packages/ckeditor5-image/docs/features/images-installation.md

```diff
@@ -26,6 +26,28 @@ import { LinkImage } from '@ckeditor/ckeditor5-link';
 ClassicEditor
 	.create( document.querySelector( '#editor' ), {
 		plugins: [ Image, ImageToolbar, ImageCaption, ImageStyle, ImageResize, LinkImage ],
+		toolbar: [ 'insertImage', /* ... */ ],
```

Comment:
The `insertImage` component is registered by the `ImageInsertUI` plugin and that plugin is required only by:
* `ImageInsert` plugin (it requires plugins: `ImageUpload`, `ImageInsertViaUrl`, `ImageInsertUI`)
* `ImageInsertViaUrl` plugin (it requires only `ImageInsertUI` and defines the default configuration for `image.insert.integrations`)

The URL integration is registered by the `ImageInsertUI` itself and it's using the `insertImage` and `replaceImageSource` commands (both registered by the `ImageEditing` plugin).

This looks like we might need to make the `ImageInsertUI` the dependency of some other common image plugin to make this available for the broader audience without the requirement to include the insert by URL plugin.

Reply:
How about `Image` itself? That's the top-level stuff then and is basically (almost) always there.